Ml with radical

alexanderhalpern opened the pull request.

arupcsedu commented:
Can you move all files to the radical-dl directory?
Remove unnecessary source files. Like hello_world files.

alexanderhalpern commented:
Yes, removed those files. I also created a folder specifically for cnn-training for better organization down the line but let me know if I should remove the folder. I also added a README so others can follow along.

arupcsedu commented:
you can basically drop the .sh file and use this task description:
```
        td = rp.TaskDescription()
        td.executable     = 'python3'
        td.arguments      = 'single-gpu-cnn.py '      \
                            '--epochs 10 '            \
                            '--batch-size 64 '        \
                            '--test-batch-size 1000 ' \
                            '--lr 0.01 '              \
                            '--gamma 0.7 '            \
                            '--save-model'.split()
        td.input_staging  = ['%s/single-gpu-cnn.py' % PWD]
```

OMP_NUM_THREADS will be set by RP, there is not need to figure this out yourself.
